src/test/java/com/fasterxml/jackson/databind/introspect/Transient3948Test.java

```diff
@@ -0,0 +1,92 @@
+package com.fasterxml.jackson.databind.introspect;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.BaseMapTest;
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.Serial;
+import java.io.Serializable;
+
+public class Transient3948Test extends BaseMapTest {
+
+    public static class Obj implements Serializable {
+
+        @Serial
```

Comment:
Added in JDK 14 cannot be added

Reply:
Ah. Yeah, I noticed that for me IDE can easily "upgrade" JDK to use as well, due to maybe thinking later one needed due to some profile in `pom.xml` requiring later.